models/users.js

```diff
@@ -215,6 +215,35 @@ const fetchUserImage = async (users) => {
   return images;
 };
 
+/**
+ * Add default status
+ * @returns {Promise<string[]>}
+ */
+const addDefaultStatus = async () => {
+  try {
+    const usersSnapshot = await userModel.get();
+    const users = [];
+    const addedDefaultStatusUsers = [];
+
+    usersSnapshot.forEach((doc) => users.push({ id: doc.id, ...doc.data() }));
+
+    for (const user of users) {
+      if (user.status === undefined) {
+        const userDoc = userModel.doc(user.id);
+        await userDoc.update({
+          status: "idle",
+        });
```

Comment:
No need of making `userDoc` variable since we aren't using it anywhere else
```suggestion
        await userModel.doc(user.id).update({
          status: "idle",
        });
```